x-pack/legacy/plugins/siem/public/components/drag_and_drop/draggable_wrapper.tsx

```diff
@@ -166,12 +162,8 @@ type Props = OwnProps & DispatchProps;
  * data provider associated with the item being dropped
  */
 class DraggableWrapperComponent extends React.Component<Props> {
-  public shouldComponentUpdate = ({ dataProvider, render, width }: Props) =>
-    isEqual(dataProvider, this.props.dataProvider) &&
-    render !== this.props.render &&
-    width === this.props.width
```

Comment:
the comparison of with `width` prop here should be replaced with the `truncate` prop that replaced it

Reply:
👍